api/e2e/test/router_test.go

```diff
@@ -7,8 +7,8 @@ import (
 
 	upiv1 "github.com/caraml-dev/universal-prediction-interface/gen/go/grpc/caraml/upi/v1"
 	"github.com/gavv/httpexpect/v2"
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
curious whats the motivation

Reply:
It's one of the messages returned from our linter that dot imports are bad (but it seems like not everyone [agrees](https://github.com/golang/lint/issues/179) with them); I can't remember why I would've changed this if it was just a warning and not an error - lemme try removing this commit and see if it still works.